usefulUtilities/avatarEntityRemover/avatarEntityRemover.js

```diff
@@ -0,0 +1,155 @@
+//
+//  avatarEntityRemover.js
+//
+//  Created by Zach Fox on 2019-03-15
+//  Copyright 2019 High Fidelity, Inc.
+//
+//  See accompanying README.md for usage instructions.
+// 
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+
+(function () {
+    var APPROVED_USERNAMES = ["Whyroc", "Judas", "Monoglu", "Spade_", "Red_SIM",
+        "ultranique", "Lyra121", "project8vr", "DragonSMP", "123.Basinsky.321",
+        "SUP3RFlyN1NJA", "Koala", "bon3s", "Aimily", "Saracen", "Silverfish",
+        "Snow", "ASingleGiggle", "XaosPrincess", "krougeau", "CornyNachos",
+        "bigtin", "Octuplex", "Dalek", "MMMaellon", "Dalus", "Scena", "Lathe",
+        "vegaslon", "Xeverian", "VardVR", "1029chris", "Menithal", "Virtual_Origin",
+        "Grifftech", "zfox", "scobot", "prophecy288"];
+    var removedAvatarEntityProperties = [];
+    var enableAvatarEntityRestore = false;
+    var kickDomain = "hifi://domain";
+    var enableCollisionlessAvatarEntities = false;
+    var tryAgainDeleteInterval = false;
+    var TRY_DELETE_AGAIN_MS = 1000;
+
+
```

Comment:
Does the file need comments on functions?  No worries if not, but maybe if intended for other devs, might not be bad.

Reply:
ah, thank you, good call